detections/experimental/endpoint/detect_suspicious_filenames_using_pretrained_model_in_dsdl.yml

```diff
@@ -0,0 +1,117 @@
+name: Detect suspicious processes using a pretrained model in DSDL
```

Comment:
can you update the name of the corresponding file names to match the search names?